src/Streamly/System/Sh.hs

```diff
@@ -47,9 +47,10 @@ where
 
 import Control.Monad.Catch (MonadCatch)
 import Data.Word (Word8)
+import Streamly.Data.Array.Unboxed (Array)
 import Streamly.Data.Fold (Fold)
-import Streamly.Prelude (SerialT, MonadAsync)
-import Streamly.Data.Array.Foreign (Array)
+import Streamly.Data.Stream (Stream)
+import Streamly.Data.Stream.Concurrent (MonadAsync)
```

Comment:
Is this all we need from the `streamly` package? Then we can define this locally and remove the dependency.

Reply:
We can. Let me do that.